roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -265,3 +265,38 @@
     executable: /bin/bash
   when:
     - '"ceph" in enable_services'
+
+- name: create devstack local conf with ironic enabled
+  shell:
+    cmd: |
+      set -e
+      set -x
+      cat << EOF >> /tmp/dg-local.conf
+      enable_plugin ironic https://git.openstack.org/openstack/ironic
+      DEFAULT_INSTANCE_TYPE: baremetal
+      OVERRIDE_PUBLIC_BRIDGE_MTU: 1400
+      VIRT_DRIVER=ironic
+      BUILD_TIMEOUT=720
```

Comment:
Note: We only use 720 seconds in OpenStack CI due to noisy neighbors. 600 should be fine in most cases.

Reply:
Thanks @juliakreger I think we can go with this and see how it will work on gophercloud so we can tune the values.